vladiate/vlad.py

```diff
@@ -166,6 +177,19 @@ def validate(self):
                             self.failures[field_name][line].append(e)
                             self.invalid_lines.add(self.line_count)
                             validator.fail_count += 1
+                if self.total_lines != 0:
+                    if (
+                        validator.fail_count / self.total_lines
+                        > self.file_validation_threshold
+                    ):
+                        self.logger.error(
+                            "  {} failed {} time(s) ({:.1%})".format(
+                                validator.__class__.__name__,
+                                validator.fail_count,
+                                validator.fail_count / self.line_count,
+                            )
+                        )
+                        return False
```

Comment:
```suggestion
                if (
                    self.file_validation_threshold
                    and self.total_lines > 0
                    and validator.fail_count / self.total_lines
                    > self.file_validation_threshold
                ):
                    self.logger.error(
                        "  {} failed {} time(s) ({:.1%})".format(
                            validator.__class__.__name__,
                            validator.fail_count,
                            validator.fail_count / self.line_count,
                        )
                    )
                    return False
```

Reply:
Thank you for the suggestion. I have now updated the condition.